module/cmd/gravity/cmd/genaccounts_test.go

```diff
@@ -69,7 +74,7 @@ func TestAddGenesisAccountCmd(t *testing.T) {
 			ctx = context.WithValue(ctx, client.ClientContextKey, &clientCtx)
 			ctx = context.WithValue(ctx, server.ServerContextKey, serverCtx)
 
-			cmd := simcmd.AddGenesisAccountCmd(home)
+			cmd := AddGenesisAccountCmd(home)
```

Comment:
Why does this need to be changed from the `simcmd` method? The test passes without this change.

Reply:
It was testing `AddGenesisAccountCmd` from sdk, which already covered by its own [test](https://github.com/cosmos/cosmos-sdk/blob/3a7a2645f05f0aecdf96ebda14a7c62f6d4ddc55/x/genutil/client/cli/genaccount_test.go#L25)